lib/javascript/JavascriptParser.js

```diff
@@ -2408,7 +2369,14 @@ class JavascriptParser extends Parser {
 						expression,
 						exprInfo.name,
 						exprInfo.rootInfo,
-						members
+						members.slice(),
```

Comment:
Not sure, maybe better slice on line 2378

Reply:
That doesn't work because it's in a closure and called later. walkMemberExpressionWithExpressionName modifies members that why we need a copy.

Reply:
ah yes `walkMemberExpressionWithExpressionName` modifies it..